rust/src/lib.rs

```diff
@@ -207,15 +225,17 @@ impl Toodle {
                         [?l :label/name ?name]
                         [?l :label/color ?color]
         ]"#;
-        self.connection
-            .query_args(query, vec![(Variable::from_valid_name("?item_uuid"), item_uuid.to_typed_value())])
+        let in_progress_read = self.connection.begin_read().expect("expected a read transaction");
```

Comment:
Don't use `expect` for these; that's just `unwrap` with a nice message! You're in a function that returns `Result`, so just `?`. If that doesn't work, you need to add a `From` implementation somewhere.

Reply:
All the others were `?` so I think I just missed this one.